cmd/skopeo/main.go

```diff
@@ -54,6 +56,12 @@ func createApp() *cli.App {
 		if c.GlobalBool("debug") {
 			logrus.SetLevel(logrus.DebugLevel)
 		}
+		if c.GlobalIsSet("cert-path") {
+			logrus.Warn("'--cert-path' is deprecated, please set this on the specific subcommand")
+		}
+		if c.GlobalIsSet("tls-verify") {
+			logrus.Warn("'--tls-verify' is deprecated, please set this on the specific subcommand")
```

Comment:
… assuming that we would silently ignore them; as long as we do use the values, we should only warn of course.